.github/workflows/bootstrap.yml

```diff
@@ -84,7 +84,7 @@ jobs:
     runs-on: ${{ matrix.runner }}
     strategy:
       matrix:
-        runner: ['macos-13', 'macos-14', "ubuntu-latest"]
+        runner: ['macos-14', "ubuntu-latest"]
```

Comment:
Thanks for the PR. Can you:
- substitute `macos-13` with `macos-15-intel`
- substitute `macos-14` with `macos-latest`

instead?

Reply:
@alalazo PS: `-latest` is not a good idea because CI should have as few moving pieces as possible, ideally zero. Moving pieces break pull requests in just the wrong moment.

Reply:
@alalazo pushed.